### PR TITLE
Update installation pages for 1.16

### PR DESCRIPTION
**Pull request by ccedric**
Modifications:
- updated configuration screenshots
- Mention that we now support ARM

ref: MISC-284

I updated the adminCredentials.png image to the on-premises one, as I don't think many marketplace users read this part of the doc

### CODE REVIEW

File: content/reference/current/installation/manual/index.md [hunk 1]
```diff
@@ -19,13 +19,11 @@ weight: 102
...
-
-We don't support ARM at the moment.
+We support both AMD64 and ARM for both Gatling Enterprise and Gatling injectors.
 
 ## JDK
 
-Gatling Enterprise components runs on a JVM and requires a modern Hotspot-based JDK 8 or 11.
+Gatling Enterprise components runs on a JVM and requires a modern Hotspot-based JDK 8, 11 or 17.
 
 We recommend you use JDK builds from [AdoptOpenJDK](https://adoptopenjdk.net/).
```
Comment (tpetillot) [inline content/reference/current/installation/manual/index.md:28]: Don't we recommend azul now?
```suggestion
We recommend you use JDK builds from [Azul System's Zulu](https://www.azul.com/downloads/?package=jdk).
```

File: content/reference/current/installation/manual/index.md [hunk 2]
```diff
@@ -445,7 +443,7 @@ By default, all registered users can connect as a global viewer and need an admi
 
 - Go to "App registrations"
 - Click on "New registration"
-- Set the redirect URI to http://your-frontline-url:your-frontline-port/redirected
+- Set the redirect URI to http://your-gatling-enterprise-url:your-gatling-enterprise-port/redirected
```
Comment (tpetillot) [inline content/reference/current/installation/manual/index.md:446]:
```suggestion
- Set the redirect URI to https://your-gatling-enterprise-url:your-gatling-enterprise-port/redirected
```

File: content/reference/current/installation/manual/index.md [hunk 3]
```diff
@@ -473,7 +471,7 @@ oidc {
 - Navigate to admin => security => api => authorization server
 - Make sure the openid, profile and email scopes are present
 - Create an OpenId app
-- Set the login redirect URI to http://your-frontline-url:your-frontline-port/redirected
+- Set the login redirect URI to http://your-gatling-enterprise-url:your-gatling-enterprise-port/redirected
```
Comment (tpetillot) [inline content/reference/current/installation/manual/index.md:474]:
```suggestion
- Set the login redirect URI to https://your-gatling-enterprise-url:your-gatling-enterprise-port/redirected
```

File: content/reference/current/installation/manual/index.md [hunk 4]
```diff
@@ -19,13 +19,11 @@ weight: 102
...
 
-We currently only support AMD64 for both Gatling Enterprise and Gatling injectors.
-
-We don't support ARM at the moment.
+We support both AMD64 and ARM for both Gatling Enterprise and Gatling injectors.
 
 ## JDK
 
-Gatling Enterprise components runs on a JVM and requires a modern Hotspot-based JDK 8 or 11.
+Gatling Enterprise components runs on a JVM and requires a modern Hotspot-based JDK 8, 11 or 17.
```
Comment (guilgaly) [inline content/reference/current/installation/manual/index.md:26]:
```suggestion
Gatling Enterprise components run on a JVM and require a modern Hotspot-based JDK 8, 11 or 17.
```

**Pull request merged**